fbw-a32nx/src/base/flybywire-aircraft-a320-neo/SimObjects/AirPlanes/FlyByWire_A320_NEO/model/A320_NEO_INTERIOR.xml

```diff
@@ -360,12 +360,12 @@
                 <POTENTIOMETER>87</POTENTIOMETER>
             </UseTemplate>
 
-            <UseTemplate Name="ASOBO_LIGHTING_Potentiometer_Emissive_Template">
-                <NODE_ID>SCREEN_AUTOPILOT</NODE_ID>
-                <POTENTIOMETER>87</POTENTIOMETER>
-                <!-- ref FBW-22-06 C/B 5LF -->
-                <EMISSIVE_CODE>(L:A32NX_ELEC_AC_1_BUS_IS_POWERED, Bool)</EMISSIVE_CODE>
-            </UseTemplate>
+            <Component ID="SCREEN_AUTOPILOT" Node="SCREEN_AUTOPILOT">
+                <UseTemplate Name="ASOBO_GT_Material_Emissive_Code">
+                    <!-- ref FBW-22-06 C/B 5LF -->
+                    <EMISSIVE_CODE>(L:A32NX_ELEC_AC_1_BUS_IS_POWERED, Bool) (L:A32NX_AMBIENT_BRIGHTNESS) 0.05 * 0.005 max (A:LIGHT POTENTIOMETER:87, percent over 100) max *</EMISSIVE_CODE>
```

Comment:
Does this allow the digits to remain visible when the AC Bus is not powered depending on ambient light (RPN kills me 😅 ) ? If so LGTM

Reply:
Should it?